tcms/static/js/bugs.js

```diff
@@ -0,0 +1,49 @@
+const bugDetailsCache = {}
+
+function loadBugs(filter) {
+    $('#bugs').DataTable({
```

Comment:
this isn't going to work. You have removed the id attribute of the table when migrating to an includable template.